opengrok-web/pom.xml

```diff
@@ -124,6 +124,23 @@ Portions Copyright (c) 2018, 2020, Chris Fraire <cfraire@me.com>.
             <groupId>org.webjars.npm</groupId>
             <artifactId>showdown</artifactId>
             <version>1.9.1</version>
+            <exclusions>
+                <exclusion>
```

Comment:
why is this necessary ?

Reply:
https://www.npmjs.com/package/showdown has one dependency: `yargs` which in turn has 7 dependencies etc. and I haven't noticed that the dependency is required for our needs -> it has worked without it before and even now :D

Reply:
I may be a good idea to write a comment there that we don't actually need any dependencies from it. After a year man can be surprised what was the intention here.

Reply:
done